src/Pages/Recipe/Recipe.js

```diff
@@ -0,0 +1,167 @@
+    });
+    return recipe;
+  };
+
+  useEffect(() => {
+    fetchRecipe()
+      .then((response) => {
+        setData(response.data);
+        setIngredients(response.data.ingredients);
+        setInstructions(response.data.instructions);
+        setIsFav(response.data.isFavorite);
+      })
+      .catch((error) => console.log(error));
+  }, []);
+
+  const add = async () => {
+    const favRecipe = {
+      recipeId: data.id,
+      userId: "",
+      title: data.title,
+      image: data.image,
+    };
+
+    try {
+      const { recipe } = await axios.post("/api/v1/recipes/list/", favRecipe, {
+        headers: {
+          Authorization: "Bearer " + token,
+          "Content-Type": "application/json",
+          Accept: "application/json",
+        },
+      });
+      setFavorite(favRecipe);
+      setIsFav(true);
+    } catch (err) {
+      setErr(err.message);
+    }
+  };
+
+  const remove = async () => {
+    try {
+      const { recipe } = await axios.delete(
+        `/api/v1/recipes/${params.id}`,
+
+        {
+          headers: {
+            Authorization: "Bearer " + token,
+            "Content-Type": "application/json",
+            Accept: "application/json",
+          },
+        }
+      );
+      setIsFav(false);
+    } catch (err) {
+      setErr(err.message);
+    }
+  };
+
+  const sanitizedData = () => ({
+    __html: DOMPurify.sanitize(data.summary),
+  });
+
+  return (
+    <>
+      <div className="pageWrapper">
+        <div className="topContainer">
+          <img alt={data.title} src={data.image} />
+          <div className="recipeInfo">
+            <h1>{data.title}</h1>
+            <ul>
+              <li>
+                <ImSpoonKnife className="icon" />
+                {data.servings} servings.
+              </li>
+              <li>
+                <HiClock className="icon" />
+                ready in {data.readyInMinutes} mins.
+              </li>
+            </ul>
+            <div>
+              {isFav ? (
+                <Button
+                  color="error"
+                  type="submit"
+                  size="large"
+                  onClick={remove}
+                >
+                  <FavoriteIcon />
+                </Button>
+              ) : (
+                <Button color="error" type="submit" size="large" onClick={add}>
+                  <FavoriteBorderIcon /> Save to favorite
+                </Button>
+              )}
+
+              <NutritionModal />
+            </div>
+          </div>
+        </div>
+
+        <div className="ingredientsTable"></div>
+        <h2>Ingredients</h2>
+        <div className="nutrition">
+          <ul>
+            {ingredients.map((i) => {
+              return (
+                <div className="ingredients" key={i}>
+                  <MdOutlineCircle color="green" />
+                  <li>{i}</li>
+                </div>
+              );
+            })}
+          </ul>
+        </div>
+        <h2>Instructions</h2>
```

Comment:
The instructions'  bulleted point does not look good. I think that similar Ingredients' bulleted is good. 
<img width="160" alt="Screen Shot 2022-11-27 at 9 27 01 PM" src="https://user-images.githubusercontent.com/88150284/204200697-ca2b874e-11fa-45ba-bb74-189a0a19735d.png">